OpenRobertaServer/staticResources/index.html

```diff
@@ -261,7 +261,7 @@ <h2 id="tutorial-header"></h2>
         </div>
         <!-- Roberta-Logo -->
         <div class="logo">
-            <img href="#" id="img-nepo" class="img-nepo" src="/css/svg/NEPO_Logo_RGB.svg" />
+            <img href="#" id="img-nepo" class="img-nepo" src="css/svg/NEPO_Logo_RGB.svg" alt="Cannot display Image"/>
```

Comment:
I would recommend to have "NEPO Logo" here

Reply:
Do you mean as an alternative (`alt`)?

Reply:
Yes, the alt tag is e.g. also the information that screen reader uses, so the text should be an alternative of the image itself

Reply:
This is done.

Reply:
Can you marge this?

Reply:
There are many changes in the code, where we are not sure if it will work in all circumstances (all browsers and systems). You also have changed a lot of css like padding etc. This will change the layout and maybe confuse the user. Can you explain your Intention for this?

Reply:
Most of them are duplicates and others are where we can merge multiple properties into a single one.
This did not change any layout as I have tested them on the below browsers.
- `padding` default is 0
- `box-shadow` default value is none
- A simple integer does not mean anything, it's better to change them to px.